reference/fleet/install-elastic-agents.md

```diff
@@ -68,6 +68,52 @@ If you are using {{agent}} with [{{serverless-full}}](/deploy-manage/deploy/elas
+
+{{agent}} distribution packages are offered in multiple flavors. This gives you control over the set of components included in the package and the size of the package footprint when it's installed.
+
+### Flavors for regular package installs
+
+#### Basic flavor [elastic-agent-basic-flavor-regular]
+
+The basic {{agent}} flavor is installed by default using the `elastic-agent install` command with an agent package: `.zip`, .`tar.gz`, `.deb`, `.rpm`, or `.msi`. This flavor includes only the following components:
+
+* `agentbeat` - used to implement all [{{beats}}](beats://reference/index.md)-based integrations.
+* `endpoint-security` - used to implement [{{elastic-defend}}](../../solutions/security/configure-elastic-defend.md).
+* `pf-host-agent` - used to collect profiling data from hosts as part of Elastic [Universal Profiling](../../solutions/observability/infra-and-hosts/get-started-with-universal-profiling.md).
+
+This basic package is suitable for most use cases and it offers a reduced size on disk.
+
+#### Servers flavor [elastic-agent-servers-flavor-regular]
+
+The servers {{agent}} flavor is installed using the `elastic-agent install --install-servers` command, or for RPM and DEB packages the `ELATIC_AGENT_FLAVOR=servers` command. In addition to components included in the basic flavor, this flavor also includes:
+
+* `apm-server` - implements the Elastic [APM Server](../../solutions/observability/apps/get-started-with-apm.md).
+* `cloudbeat` - implements [Cloud Security Posture Management (CSPM)](../../solutions/security/cloud/cloud-security-posture-management.md) integrations.
+* `fleet-server`, implements [Fleet Server](../fleet/fleet-server.md) for managing {{agents}}.
+* `pf-elastic-symbolizer` - a server side component of Elastic [Universal Profiling](../../solutions/observability/infra-and-hosts/get-started-with-universal-profiling.md).
+* `pf-elastic-collector` - a server side component of Elastic [Universal Profiling](../../solutions/observability/infra-and-hosts/get-started-with-universal-profiling.md).
+
+Beginning in version 9.0, for {{agents}} to have the full functionality that was supported by default in pre-9.x versions, you need to install the servers {[agent]} flavor.
+
+### Flavors for container package installs 
+
+#### Basic flavor [elastic-agent-basic-flavor-container]
+
+For containerized environments, the basic {{agent}} flavor is installed using the `elastic-agent-slim` command with an agent container package. This flavor contains the same set of components described in [Basic flavor](#elastic-agent-basic-flavor-regular) above.
+
+#### Servers flavor [elastic-agent-servers-flavor-container]
+
+For containerized environments, the servers {{agent}} flavor is installed using the default `elastic-agent` command with an agent container package. This flavor contains the same set of components described in [Servers flavor](#elastic-agent-servers-flavor-regular) above.
+
+#### Wolfi flavor [elastic-agent-wolfi-flavor]
```

Comment:
This isn't a flavor it's a different base image. The actual functionality is unchanged. Base images vs flavors are conceptually separate.

Flavors = "what does the image contain"
Base Image = "what Linux variant is it" where wolfi has the least amount of Linux possible to minimize exposure to CVEs.

Reply:
The Wolfi change is stack wide and not specific to agent or beats, I hope there is some existing stack documentation on the introduction of the wolfi image and switch from Ubuntu to UBI for the plain elastic-agent container you can just link to instead of writing it from scratch. Every product has made this change.

Reply:
OK I see some of this below, but it doesn't cover that the elastic-agent image was switched from Ubuntu to RedHat UBI in 9.0. 

IIRC the primary limitation of Wolfi is you can't manually extend or customize the image by building a new image from it without worrying about Chainguard licensing.

Reply:
> I hope there is some existing stack documentation on the introduction of the wolfi image and switch from Ubuntu to UBI for the plain elastic-agent container you can just link to instead of writing it from scratch.

Yup. Details are in my comment further up, but the first mention of the Wolfi images has a link to Maxine's detailed blog, and I've added a note about the Ubuntu -> UBI conversion for 9.0, with a link to Panos' PR for details.